src/build.xml

```diff
@@ -19,7 +21,7 @@
 
 <Target Name="RestorePackages">
   <Exec Condition="!Exists($(NuGetPath))" Command="powershell.exe -ExecutionPolicy ByPass -Command Invoke-WebRequest -Uri https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -OutFile $(NuGetPath)" />
-  <Exec Command="dotnet restore $(SolutionNET6) -p:Platform=NET60_Windows --runtime=win10-x64"/>
+  <Exec Command="dotnet restore $(Solution) -p:Platform=&quot;$(Platform)&quot; --runtime=$(RuntimeIdentifier)"/>
```

Comment:
do you need then `&quot;` escaping?

Reply:
Platform now contain a space. But that argument is probably not needed any more.